From 6bc8fbda610c6ce0a7c20242ff7ed811cdced89d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@hoyer.xyz>
Date: Thu, 12 Oct 2017 11:23:38 +0200
Subject: [PATCH] Merge pull request #223 from
 bmr-cymru/bmr-ignore-activation-skip

lvm_scan.sh: use -K when calling lvchange
---
 modules.d/90lvm/lvm_scan.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90lvm/lvm_scan.sh b/modules.d/90lvm/lvm_scan.sh
index 8bf08641..61a18b24 100755
--- a/modules.d/90lvm/lvm_scan.sh
+++ b/modules.d/90lvm/lvm_scan.sh
@@ -114,7 +114,7 @@ if [ -n "$LVS" ] ; then
     info "Scanning devices $lvmdevs for LVM logical volumes $LVS"
     lvm lvscan $lvm_ignorelockingfailure 2>&1 | vinfo
     for LV in $LVS; do
-        lvm lvchange --yes -ay $lvm_quirk_args $LV 2>&1 | vinfo
+        lvm lvchange --yes -K -ay $lvm_quirk_args $LV 2>&1 | vinfo
     done
 fi
 

